src/librustc/session/mod.rs

```diff
@@ -431,8 +431,9 @@ impl Session {
                     diag_builder.span_note(span, message);
                 }
                 DiagnosticBuilderMethod::SpanSuggestion(suggestion) => {
-                    let span = span_maybe.expect("span_suggestion needs a span");
-                    diag_builder.span_suggestion(span, message, suggestion);
+                    let span = span_maybe.expect("span_suggestion_* needs a span");
+                    diag_builder.span_suggestion_with_applicability(span, message, suggestion,
+                                                                    Applicability::Unspecified);
```

Comment:
Change formatting to have each argument in its own line.

Reply:
`DiagnosticBuilderMethod::SpanSuggestion` should probably carry the `Applicability` level, but that is outside the scope of this PR.